osu.Game/Overlays/Mods/BpmDisplay.cs

```diff
@@ -0,0 +1,209 @@
+                                    {
+                                        contentBackground = new Box
+                                        {
+                                            RelativeSizeAxes = Axes.Both
+                                        },
+                                        new OsuSpriteText
+                                        {
+                                            Anchor = Anchor.Centre,
+                                            Origin = Anchor.Centre,
+                                            Margin = new MarginPadding { Horizontal = 18 },
+                                            Shear = new Vector2(-ShearedOverlayContainer.SHEAR, 0),
+                                            Text = "Average BPM",
+                                            Font = OsuFont.Default.With(size: 17, weight: FontWeight.SemiBold)
+                                        }
+                                    }
+                                },
+                                new Container
+                                {
+                                    Anchor = Anchor.Centre,
+                                    Origin = Anchor.Centre,
+                                    Shear = new Vector2(-ShearedOverlayContainer.SHEAR, 0),
+                                    AutoSizeAxes = Axes.Both,
+                                    Children = new Drawable[]
+                                    {
+                                        counter = new BpmCounter
+                                        {
+                                            Current = { BindTarget = current }
+                                        },
+                                        dashText = new OsuSpriteText
+                                        {
+                                            Anchor = Anchor.Centre,
+                                            Origin = Anchor.Centre,
+                                            Font = OsuFont.Default.With(size: 17, weight: FontWeight.SemiBold),
+                                            Text = "-"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            contentBackground.Colour = colourProvider.Background4;
+        }
+
+        protected override void LoadComplete()
+        {
+            mods.BindValueChanged(m =>
+            {
+                settingChangeTracker?.Dispose();
+
+                refresh();
+
+                settingChangeTracker = new ModSettingChangeTracker(m.NewValue);
+                settingChangeTracker.SettingChanged += _ => refresh();
+            }, true);
+            working.ValueChanged += _ => refresh();
```

Comment:
Nitpick, but why is one of these using `BindValueChanged(...)` and one is doing `ValueChanged += ...`?